uwblueprint/marillac-place
# Develop frontend for creating announcements

## Notion ticket link

[Develop frontend for creating announcements](https://www.notion.so/uwblueprintexecs/Develop-frontend-for-creating-announcements-65bf4e8829be450986c611d93a0946f6?pvs=4)



## Implementation description
* Add room button creates a new tab with greyed out title "New Announcement" 
* Clicking on the new tab brings up a different right panel. There is a + button where you can add and remove Rooms
* Adding rooms will change the title of the tab from "New Announcement" to a list of rooms 
* Pressing post with rooms defined and text in the text box will remove the new announcement



## Steps to test
1. Click add room button
2. Click on New Announcements tab
3. Try adding and removing rooms (tab title will also change)
4. Add text in "Type an announcement..." and press post. This should delete the new tab. 



## What should reviewers focus on?
*


## Checklist
